packages/smooth_app/lib/pages/prices/price_amount_card.dart

```diff
@@ -35,22 +35,33 @@ class _PriceAmountCardState extends State<PriceAmountCard> {
   final TextEditingController _controllerWithoutDiscount =
       TextEditingController();
 
+  @override
+  void initState() {
+    super.initState();
+    _controllerPaid.text = _model.paidPrice;
```

Comment:
Hmm, I've double-checked and I can't see how this could happen:
* the only setter in this line is `_controllerPaid.text`
* `.text` doesn't trigger anything, in particular not `onChanged`
* besides, it's in the `initState`, when the widget isn't even built, so no `setState` expected.

That said, if you prefer a safer way (reviewing-wise) to do exactly the same thing, would you like me to rewrite it like that:
```dart
  late final TextEditingController _controllerPaid;
  late final TextEditingController _controllerWithoutDiscount;

  @override
  void initState() {
    super.initState();
    _controllerPaid = TextEditingController(text: _model.paidPrice);
    _controllerWithoutDiscount =
        TextEditingController(text: _model.priceWithoutDiscount);
  }
```

Reply:
A `TextEditingController` is a `ValueNotifier` and text is linked the `value`.
Hence the dependency to listeners